packages/testing/config/storybook/main.js

```diff
@@ -14,6 +14,28 @@ const config = getConfig()
 
 const rwjsPaths = getPaths()
 
+function isPackageInstalled(alias) {
+  try {
+    return Boolean(require(alias))
+  } catch (e) {
+    return false
+  }
+}
+
+function withEmotionVersionFallback(config) {
```

Comment:
This is our proposed solution coming soon in 6.4 FWIW: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#emotion11-quasi-compatibility

Testing/feedback very welcome! 🙏

Reply:
Awesome! I configured the alias according to https://github.com/storybookjs/storybook/blob/200902945a60042584fbcb01c7f0af959e9fada0/lib/theming/paths.js and now it works :tada: